.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+# Default Settings
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
Would you be opposed to 4 spaces instead of two? This is the standard I've gotten used to from most IDEs/text editors.

Reply:
I have no problem with using 4 space indent as the default 👍

Reply:
The default indent size has been updated.